server/webhook.go

```diff
@@ -189,7 +189,7 @@ func (p *Plugin) GetWebhookURL(jiraURL string, teamID, channelID string) (subURL
 
 	v := url.Values{}
 	v.Add("secret", cf.Secret)
-	subURL = p.GetPluginURL() + instancePath(routeAPISubscribeWebhook, instanceID) + "?" + v.Encode()
+	subURL = p.GetPluginURL() + instancePath(routeAPI+routeAPISubscribeWebhook, instanceID) + "?" + v.Encode()
```

Comment:
I suppose we can also have a `makeAPIRoute`

Reply:
Updated